src/toolkit/res_buf.h

```diff
@@ -85,6 +95,13 @@ class ResBuf {
     cap_ = cap;
   }
 
+  /// Sets whether the buffer should keep packaged resources
+  void keep_packaging(bool keep_packaging) {
+    keep_packaging_ = keep_packaging;
+  }
```

Comment:
After the discussion of making sure this is consistent with `is_bulk_`, I realized that we don't let anyone change `is_bulk_` after initialization, which probably makes a lot of sense.

Should we also not allow `keep_packing_` to change after initialization?  (Thus undoing everything we discussed yesterday)

Reply:
No, I feel like it's fine to do so